librz/core/cmd_descs/cmd_type.yaml

```diff
@@ -81,6 +150,28 @@ commands:
         cname: type_noreturn_del_all
         summary: Remove all noreturn references
         args: []
+  - name: to
+    summary: Load types from C header file
+    subcommands:
+      - name: to
+        cname: type_load_file
+        summary: Load types from C header file
+        args:
+          - name: file
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: toe
```

Comment:
Other places use `!` as suffix for the editor, however i think there's no real consistent way at the moment. I like `e`, but it could be some commands already have it occupied... Maybe `E`? Not sure...

Reply:
I can note it in my TODO but this is out of the scope of this PR, imho.

Reply:
It was just for discussion, to check if you have any idea about it.